agent/gather.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+source $SCRIPTDIR/agent/04_agent_configure.sh
+
+get_static_ips_and_macs
+
+if [[ "$IP_STACK" = "v4" ]]; then
+    ip=${AGENT_NODES_IPS[0]}
+  else
+    ip=${AGENT_NODES_IPSV6[0]}
+  fi
```

Comment:
This approach may certainly work, but I see a potential fragility, either because the `get_static_ips_and_macs` performs other actions either because it could change in future and have a impact here.
Since by design we decided to use the first IP calculated as the node0 ip (as correctly also adopted here), why don't we simply serialize that ip in a `node0-ip` file [when it's defined](https://github.com/openshift-metal3/dev-scripts/blob/ac7ac11880a0fe76a510fe1504ea125eca34cad4/agent/04_agent_configure.sh#L49), so that it could be reused here?